src/main/resources/additional_properties.json

```diff
@@ -18,5 +18,365 @@
+          "title": "Simple Debezium Topic Router (not enforcing message key uniqueness)",
+          "description": "Simple Debezium Topic Router for sending change events from multiple physical tables to one topic without enforcing message key uniqueness. See: <a href=\"https://debezium.io/documentation/reference/transformations/topic-routing.html\">Topic Routing section</a> in the Debezium docs.",
+          "required": ["debezium_topic_router_simple"],
+          "additionalProperties": false,
+          "properties": {
+            "debezium_topic_router_simple": {
+              "additionalProperties": false,
+              "required": ["topic_regex", "output_topic"],
+              "properties": {
+                "topic_regex": {
+                  "title": "Input topic regex",
+                  "description": "A regular expression on the topic name of each message that determines if it should be routed to a another destination topic and captures one or more groups of characters to be applied in <code>output_topic</code>",
+                  "type": "string",
+                  "format": "regex"
+                },
+                "output_topic": {
+                  "title": "Output topic",
+                  "description": "A regular expression replacement string that defines the destination topic name based on the captured groups from <code>topic_regex</code> .",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        {
+          "type": "object",
+          "title": "Unique Key Debezium Topic Router (enforcing message key uniqueness)",
+          "description": "Debezium Topic Router for sending change events from multiple physical tables to a specific topic and enforcing message key uniqueness. See: <a href=\"https://debezium.io/documentation/reference/transformations/topic-routing.html\">Topic Routing section</a> in the Debezium docs.",
+          "required": ["debezium_topic_router_unique_keys"],
+          "additionalProperties": false,
+          "properties": {
+            "debezium_topic_router_unique_keys": {
+              "additionalProperties": false,
+              "required": ["topic_regex", "output_topic", "key_field_name"],
+              "properties": {
+                "topic_regex": {
+                  "title": "Input topic regex",
+                  "description": "A regular expression on the original/input destination topic name of each message that determines if it should be routed to a another destination topic and captures one or more groups of characters to be applied in <code>output_topic</code>",
+                  "type": "string",
+                  "format": "regex"
+                },
+                "output_topic": {
+                  "title": "Output topic",
+                  "description": "A regular expression replacement string that defines the destination topic name based on the captured groups from <code>topic_regex</code> .",
```

Comment:
actually I meant "regular expression replacement string" to be one thing. usually you would write a single string that comes with the regular expression and the replacement, like `/$expression/$replacement/$modifier". Here it is split up  into two parts and it is hard to give that replacement a good name as it is not a plain replacement but a regex replacement string. Any ideas how to name it?

Reply:
Also about the "specified by" change... the previous parameter "topic_regex" does not specify the content, but only marks the groups that get captured by executing the regex. as a result you get a list of matches and those can be used in the regex replacement string.

Reply:
@rk3rn3r Yeah, Ok, I can see that I'm off base here, and I'll continue to think about it, but I have to admit that it's still hard for me to grasp the exact intent. I think that it would be helpful if you could provide an example for this **Output topic** field and the **Key field value** field.

Reply:
@roldanbob Maybe you look into the Debezium docs first? https://debezium.io/documentation/reference/transformations/topic-routing.html At least the functionality is described there. I don't have a good resource about the regular expressions related things/naming.

Reply:
@Naros, what do you think?

Reply:
How about:
```
A string that defines the destination topic name, optionally using the regular expression capture groups from the <code>topic_regex</code> evaluating to a dynamic destination topic name.
```
@roldanbob ?

Reply:
@MelissaFlinn, wdyt?

Reply:
Suggestion: 
A string that defines the destination topic name. The destination topic name can be static or dynamic.  To define a dynamic destination topic name, specify a regular expression that uses capture groups from the source topic <code>topic_regex</code>.

Reply:
Very good point @Naros! We should make clear that this can be one static topic name or dynamic using capture groups! And a good suggestion @MelissaFlinn because it describes the content/reason first before showing how possible values can be constructed. Although I find it not 100% precisely at the last part. I think I will combine both suggestions into:
```
A string that defines the destination topic name. The destination topic name can be static or dynamic. To define dynamic destination topic name(s) use the capture groups matched from <code>topic_regex</code>.
```
wdyt @MelissaFlinn  @roldanbob @Naros ?

Reply:
lgtm!